ServerCore/Pages/Shared/_Layout.cshtml

```diff
@@ -166,26 +175,46 @@
                             <span class="icon-bar"></span>
                             <span class="icon-bar"></span>
                         </button>
-                        <a asp-page="/." class="navbar-brand" data-toggle="dropdown">
+                        <a class="navbar-brand" data-toggle="dropdown" href="#">
                             @Event.Name - Admin
                         </a>
+                        <!-- TODO: Check for permissions / make partial -->

```

Comment:
Put IsEventAuthor and IsEventAdmin methods on EventSpecificPageModel?

Reply:
Good idea - made a note to myself so I can try it out :)